src/Confluent.Kafka/AdminClient.cs

```diff
@@ -635,6 +635,52 @@ private ListOffsetsReport extractListOffsetsReport(IntPtr resultPtr)
             };
         }
 
+        private ElectLeadersReport extractElectLeadersResults(IntPtr resultPtr)
+        {
+            IntPtr topicPartitionsPtr = Librdkafka.ElectLeaders_result_partitions(resultPtr, out UIntPtr topicPartitionsCountPtr);
+
+            if ((int)topicPartitionsCountPtr == 0)
+            {
+                return new ElectLeadersReport
+                {
+                    Error = new Error(ErrorCode.NoError),
+                    PartitionResults = new List<TopicPartitionError>()
+                };
```

Comment:
It is esssential as it tells us to report no Error in case we dont recieve any topicPartitions in the result and we dont have to raise exception for this case

Reply:
oh right this is still needed for checking that no error occurred